### ISSUE TITLE
The latest deprecation broke builds

### ISSUE THREAD

**Issue by alechirsch** [rwjblue/broccoli-unwatched-tree#4]
Please revert this deprecation and release as a patch. You can then add the deprecation and release on npm as a major release. This way the countless apps using this package do not break the next time they install, and the package still gets deprecated.

It broke because broccoli-source requires that you call `new UnwatchedDir`, whereas your package did not require this.

**Comment by rwjblue** [comment]
I will fix the issue around `new` usage, however I intend to leave the deprecation in place. 

Thank you for reporting!

**Comment by Kerrick** [comment]
Confirmed, our build has broken and I believe it traces back to this addon's release today.

```
Cannot set property '_directoryPath' of undefined
TypeError: Cannot set property '_directoryPath' of undefined
    at Directory (/home/travis/build/secondstreet/front-end/partner/node_modules/broccoli-source/index.js:6:23)
    at UnwatchedDir (/home/travis/build/secondstreet/front-end/partner/node_modules/broccoli-source/index.js:65:13)
    at Class.treeForAddonStyles (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-freestyle/index.js:55:38)
    at Class._treeFor (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/models/addon.js:534:33)
    at Class.treeForAddon (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/models/addon.js:679:46)
    at Class._treeFor (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/models/addon.js:534:33)
    at Class.treeFor (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/models/addon.js:494:21)
    at project.addons.map.addon (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/broccoli/ember-app.js:530:20)
    at Array.map (native)
    at EmberApp.addonTreesFor (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/broccoli/ember-app.js:528:30)
    at EmberApp._addonTree (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/broccoli/ember-app.js:926:36)
    at EmberApp._processedVendorTree (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/broccoli/ember-app.js:964:20)
    at EmberApp._processedExternalTree (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/broccoli/ember-app.js:995:21)
    at EmberApp.appAndDependencies (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/broccoli/ember-app.js:1104:23)
    at EmberApp.javascript (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/broccoli/ember-app.js:1236:28)
    at EmberApp.toArray (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/broccoli/ember-app.js:1673:10)
    at EmberApp.toTree (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/broccoli/ember-app.js:1695:30)
    at module.exports (/home/travis/build/secondstreet/front-end/partner/ember-cli-build.js:190:14)
    at Builder.setupBroccoliBuilder (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/models/builder.js:54:19)
    at new Builder (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/models/builder.js:33:10)
    at BuildTask.run (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/tasks/build.js:15:19)
    at Win.checkIfSymlinksNeedToBeEnabled.then (/home/travis/build/secondstreet/front-end/partner/node_modules/ember-cli/lib/commands/test.js:169:27)
```

**Comment by alechirsch** [comment]
@rwjblue @Kerrick I will be submitted a pull request to fix this shortly

**Comment by rwjblue** [comment]
Fixed by the changes in #5
